telebot/apihelper.py

```diff
@@ -332,6 +331,16 @@ def get_user_profile_photos(token, user_id, offset=None, limit=None):
         payload['limit'] = limit
     return _make_request(token, method_url, params=payload)
 
+def set_message_reaction(token, chat_id, message_id, reaction=None, is_big=None):
+    method_url = r'setMessageReaction'
+    payload = {'chat_id': chat_id, 'message_id': message_id}
+    if reaction:
```

Comment:
**Proposal.** 
Check if reaction is not list (single reaction) and convert it to list.
or
Check if reaction is not list and raise error with explanation.

**Reasonable.**
I suppose lot of mistakes by users because of parameter naming: "reaction".